pom.xml

```diff
@@ -171,10 +175,6 @@
         </dependency>
 
         <!-- elasticsearch -->
-        <dependency>
-            <groupId>org.elasticsearch</groupId>
-            <artifactId>elasticsearch</artifactId>
-        </dependency>
```

Comment:
why remove this ?

Reply:
because it serves no purpose with spring-data-elasticsearch already there